my_view_inc.php

```diff
@@ -75,7 +75,9 @@
 
 # @TODO cproensa: make this value configurable
 $t_recent_days = 30;
-$c_filter['recent_mod'] = filter_create_recently_modified( $t_recent_days );
+$t_recent_mod_filter = filter_create_recently_modified( $t_recent_days );
+$t_recent_mod_filter[FILTER_PROPERTY_HIDE_STATUS] = META_FILTER_NONE;
```

Comment:
filter_create_recently_modified(), when used to create a new filter, sets the date filter to the needed days values, and keeps the rest of properties as given by a new default filter.
Hide status is then returned as the default value as per configuration "hide_status_default".

We seem to support that this filter box must show all status, regardless of "hide_status_default" configuration, so this property makes sense to be modified in my-view-page for this specific usage.